### PR TITLE
tests: clean imports and add search test at resource level

### PR DESCRIPTION
**Pull request by ppanero**
this test demonstrates that https://github.com/inveniosoftware/invenio-app-rdm/issues/284 goes to invenio-records-resources level

### CODE REVIEW

File: tests/mock_module/api.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ class Record(RecordBase):
     schema = ConstantField(
         '$schema', 'http://localhost/schemas/records/record-v1.0.0.json')
 
-    index = IndexField('records-record-v1.0.0', search_alias='records')
+    index = IndexField('records-record-v1.0.0', search_alias='draftsresources-records')
```
Comment (lnielsen) [inline tests/mock_module/api.py:22]: Question: what's the reason for this change?

Comment (ppanero) [reply tests/mock_module/api.py:22]: @lnielsen The agreement we got to have the folders like:

```
draftsresources/    # equivalent to rdmrecords
    records/
        ...
    drafts/
        ...
```

To allow index/alias creation. How `invenio-search` works is that it will create:
- `draftsresources`: queries all indices
- `draftsresources -records`: queries all indices inside that folder
- `draftsresources -drafts`: same as above but for drafts

meaning it prefixes all sort of in a recursive way.

Comment (lnielsen) [reply tests/mock_module/api.py:22]: Ahh, got it :-) Slow today.... 😄

File: tests/resources/test_record_resource.py [hunk 2]
```diff
@@ -168,6 +168,23 @@ def test_publish_draft(client, input_data):
...
 
+def test_search_draft(client, input_data):
+    response = client.post(
+        "/mocks", data=json.dumps(input_data), headers=HEADERS)
+    recid = response.json['id']
+
+    assert response.status_code == 201
+    _assert_single_item_response(response)
+
+    response = client.get("/mocks?q=", headers=HEADERS)
```
Comment (fenekku) [inline tests/resources/test_record_resource.py:179]: This test is failing because we are hitting the published record endpoints and there are no published records; there is only a draft at this point. Publishing and searching is a test case already covered in invenio-records-resources, so I wouldn't worry about adding this test here. The `q=` case is (or should be) in invenio-records-resources I think.

**Comment by ppanero** [comment]
Agreeing with @fenekku and taking into account said test is already in invenio-records-resources I am closing this PR and porting the search alias fix to https://github.com/inveniosoftware/invenio-drafts-resources/pull/63
